server/data/mappers/educationAndTrainingMapper.test.ts

```diff
@@ -0,0 +1,107 @@
+import moment from 'moment/moment'
+import type { InductionResponse } from 'educationAndWorkPlanApiClient'
+import type { EducationAndTraining } from 'viewModels'
+import toEducationAndTraining from './educationAndTrainingMapper'
+import {
+  aLongQuestionSetInduction,
+  aShortQuestionSetInduction,
+} from '../../testsupport/inductionResponseTestDataBuilder'
+
+describe('educationAndTrainingMapper', () => {
```

Comment:
These tests are identical to what we originally had with the original mappers that map from `CiagInduction` - ie. the scenario and expected outcomes are the same, and so are the inputs in that they are either short or long question sets (except they are now `InductionResponse`s rather than `CiagInduction`s)